app-metrics/prometheus-podman-exporter/prometheus-podman-exporter-1.4.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module systemd
+
+DESCRIPTION="Exporter for podman giving containers, pods, images, volumes & networks metrics"
+HOMEPAGE="https://github.com/containers/prometheus-podman-exporter"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/prometheus-podman-exporter.git"
+else
+	SRC_URI="https://github.com/containers/prometheus-podman-exporter/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+# Main package
+LICENSE="Apache-2.0"
+# Dependencies
+LICENSE+=" BSD-2 BSD ISC MIT MPL-2.0"
+SLOT="0"
+RESTRICT="test"
```

Comment:
Please add a comment stating why tests are restricted. Link to upstream issue if there's one open.

Reply:
done